src/components/filterSelectButton/index.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { FC } from 'react';
+import { I18nContext } from '@smg-automotive/i18n-pkg';
+import {
+  ButtonGroup,
+  chakra,
+  Button as ChakraButton,
+  IconButton,
+  Popover,
+  PopoverTrigger,
+  useDisclosure,
+} from '@chakra-ui/react';
+
+import TranslationProvider from '../translationProvider';
+import { ChevronDownSmallIcon, CloseIcon } from '../icons';
+
+import { FilterSelectButtonProps } from './props';
+import FilterPopover from './Popover';
+
+export const FilterSelectButton: FC<FilterSelectButtonProps> = ({
+  actionButton,
+  displayValue,
+  initialPopoverState = 'closed',
+  isApplied,
+  label,
+  language,
+  numberOfAppliedFilters,
+  onPopoverClose,
+  onPopoverOpen,
+  onResetFilter,
+  children,
+}) => {
+  const { onOpen, onClose, isOpen } = useDisclosure({
+    defaultIsOpen: initialPopoverState === 'open',
+    onOpen: onPopoverOpen,
+    onClose: onPopoverClose,
+  });
+
+  const appliedOrOpenColorScheme = {
+    backgroundColor: 'gray.900',
+    ...(isApplied
+      ? {
+          _hover: {
+            backgroundColor: 'black',
+          },
+        }
+      : {
+          _groupHover: {
+            backgroundColor: 'black',
+          },
+        }),
+    color: 'white',
+  };
+
+  const defaultColorSchema = {
+    backgroundColor: 'gray.100',
+    _groupHover: {
+      backgroundColor: 'gray.200',
+    },
+    color: 'gray.900',
+  };
+
+  return (
+    <TranslationProvider language={language} scopes={['filterSelectButton']}>
+      <I18nContext.Consumer>
+        {({ t }) => (
+          <Popover
+            returnFocusOnClose={true}
+            placement="bottom-start"
+            isLazy={true}
+            isOpen={isOpen}
+            onOpen={onOpen}
+            onClose={onClose}
+          >
+            <ButtonGroup isAttached={true} w="full" maxW="full">
+              <PopoverTrigger>
+                <ChakraButton
+                  borderRadius="sm"
+                  borderRightColor={isApplied ? 'white' : undefined}
+                  borderRightWidth={isApplied ? '1px' : undefined}
```

Comment:
Can't we use one of our token here?

Reply:
it is a token :D https://main-components-pkg.branch.autoscout24.dev/?path=/story/foundations-theme--border